### PR TITLE
Update to final OKRs for Q1 2018

### PR DESCRIPTION
**Pull request by lidel**
This PR updates markdown copy to the version finalized during Friday's call  between 
@lidel, @olizilla  and @flyingzumwalt

### CODE REVIEW

**Comment by diasdavid** [comment]
@lidel you might want to replace this doc by a link to the OKR spreadsheet now

**Comment by lidel** [comment]
@diasdavid  just to be sure, remove it and make link in README point at spreadsheet?

**Comment by diasdavid** [comment]
@lidel exactly 👍

**Comment by flyingzumwalt** [comment]
Sorry - I wrote a comment on this yesterday but must not have submitted. 

The OKRs spreadsheet is primarily a tool for _internal_ communication across teams. It is not designed to be a means of communicating to the broader world. I think @lidel and @olizilla are doing the right thing by having a separate page that lists their working groups' OKRs, or at least the top-level Objectives. This gives you a place to provide context and explanation for outside observers and people who are relying on this work.  

To emphasize this, you might want to rename this document to ROADMAP.md and write an explanation of what OKRs are and what they tell you about the working group's "roadmap" for Q1.

**Comment by diasdavid** [comment]
@flyingzumwalt I had in mind that we would be sharing the spreadsheet with a community. Are you suggesting that every WG should create a _public_ version of their tab in the spreadsheet?

**Comment by lidel** [comment]
@flyingzumwalt  I reframed OKR document into a wider ROADMAP in which OKRs are only a quaterly-updated section ([preview](https://github.com/ipfs/in-web-browsers/blob/d441571757552f755e98a0dc62cc2d7554fc0ee7/ROADMAP.md)).  The ROADMAP itself is just a starting point, we will improve it along the way. 

Ok to merge?

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# ROADMAP for IPFS in Web Browsers
```
Comment (olizilla) [inline ROADMAP.md]: No need to SHOUT ;-)

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+- *DONE:* Create browser extension ([ipfs-companion](https://github.com/ipfs-shipyard/ipfs-companion)) that detects IPFS resources and redirects requests to local gateway.
+
+### Long Game
+- *IN PROGRESS:* Improve extension so it handles distributed protocols natively without need for third party software.
+- *IN PROGRESS:* Define the APIs, address schemes, security models, etc.
+- *IN PROGRESS:* Make the case for IPFS & Decentralized Web Protocols in Web Browsers with Use Cases, Sample Applications, Demos and Delightful Documentation
+- *IN PROGRESS:* Work with browser vendors on adding missing APIs and solving discovered problems.
+
+
+# Quarterly Objectives and Key Results
```
Comment (olizilla) [inline ROADMAP.md]: I'd drop the Q1 and Q2 2018 headers and just collapse this into a single

`Objectives and Key Results - Q1 2018` as a h2

The content is all good.

**Comment by flyingzumwalt** [comment]
👍 this looks great! Maybe we should recommend this approach to other WGs.

File: ROADMAP.md (hunk 2 above)
Comment (lidel) [reply ROADMAP.md]: Question is: are we going to replace this section every quarter or  do we want to keep old records and grow it every quarter?

Comment (olizilla) [reply ROADMAP.md]: I lean slightly towards "replace" as this isn't a changelog, but more a statement of what we're focusing on currently. Might be cool to be able to read the the full list somewhere in a couple of years time, but that could be a separate doc.

Comment (lidel) [reply ROADMAP.md]: Ok, sounds good. I removed Q2 items for now, but kept header names.
We will revisit this when OKRs for Q2 are in.

**Pull request merged**